### DISCUSSION TITLE
LAMMPS Precision

### DISCUSSION THREAD

**Discussion by npiroozan** [deepmodeling/deepmd-kit#2964] [Q&A]
Hello,

I hope you are doing well.  Forgive me for this question, as I know it has been asked before.  I'm seeing a problem that I don't understand.

To use LAMMPS in single precision, the general strategy is to change the export LAMMPS_PLUGIN_PATH to something like:

/some/workspace/deepmd/source/lib/deepmd_lmp_low

However, this doesn't work for me.  Still, I see the use of double preicison in all situations.

I built the C++ of DeePMD, and build LAMMPS with built-in mode using the exact procedure specified in the manual.

If it's not too much trouble, could you please help me with this?  Thank you very much.

**Comment by njzjz** [comment]
LAMMPS always uses double precision that is hardcoded in the LAMMPS code.
For the above reason, we have deprecated deepmd_lmp_low, which was used to control the data type of DeePMD-kit interface.

**Reply by npiroozan** [reply]
Thank you for the information. I didn't know deepmd_lmp_low had been phased out.
